shared/AppInsightsCommon/src/Telemetry/Exception.ts

```diff
@@ -745,7 +746,7 @@ export function _createExceptionDetails(logger: IDiagnosticLogger, exception: Er
 
         typeName = dataSanitizeString(logger, _getErrorType(error)) || strNotSpecified;
         message = dataSanitizeMessage(logger, _formatMessage(exception || error, typeName)) || strNotSpecified;
-        const stack = exception[strStackDetails] || _getStackFromErrorObj(exception);
+        const stack = (exception as any)[strStackDetails] || _getStackFromErrorObj(exception);
```

Comment:
Do we want to make types for generic exceptions later/plan to clean up some of these `any`s? Or is this considered fine in the web SDK?